src/Workspaces/Core/Portable/Workspace/IWorkspaceConfigurationService.cs

```diff
@@ -34,19 +34,20 @@ internal sealed class DefaultWorkspaceConfigurationService() : IWorkspaceConfigu
 [DataContract]
 internal readonly record struct WorkspaceConfigurationOptions(
     [property: DataMember(Order = 0)] SourceGeneratorExecutionPreference SourceGeneratorExecution = SourceGeneratorExecutionPreference.Automatic,
-    [property: DataMember(Order = 1)] bool ValidateCompilationTrackerStates =
+    [property: DataMember(Order = 1)] bool ReloadChangedAnalyzerReferences = true,
+    [property: DataMember(Order = 2)] bool ValidateCompilationTrackerStates =
 #if DEBUG // We will default this on in DEBUG builds
         true
 #else
         false
 #endif
     )
 {
-    public static readonly WorkspaceConfigurationOptions Default = new();
+    public static readonly WorkspaceConfigurationOptions Default = new(SourceGeneratorExecution: SourceGeneratorExecutionPreference.Automatic);
```

Comment:
Why is this being set now when it was the default already?

Reply:
So, previously, 'new()' was creating a default instance (since these are structs).  It's a pain point here.  I'm going to just make these classes acutally.  Being a struct doesn't really help us here.

Reply:
moved to class.